plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSnapshots.java

```diff
@@ -74,4 +345,20 @@ private List<Long> getSnapshotsIdsByCreationOrder(String tableName)
                 .map(row -> (Long) row.getField(ID_FIELD))
                 .collect(toList());
     }
+
+    private List<MaterializedRow> getSnapshotSummariesByCreationOrder(String tableName)
+    {
+        return getQueryRunner().execute(
+                format("SELECT CAST(SUMMARY as JSON) FROM \"%s$snapshots\" ORDER BY committed_at", tableName))
+                .getMaterializedRows();
+    }
+
+    private List<QueryId> getQueryIdsFromSnapshotSummariesByCreationOrder(String tableName)
+    {
+        return getQueryRunner().execute(
+                format("SELECT json_extract_scalar(CAST(SUMMARY AS JSON), '$.%s') FROM \"%s$snapshots\"", TRINO_QUERY_ID_NAME, tableName))
```

Comment:
lowercase `json` and `summary`
(we uppercase SQL keywords and type names in typed literals only)